omnisafe/algorithms/off_policy/ddpg.py

```diff
@@ -197,9 +197,9 @@ def _init_log(self) -> None:
         self._logger.setup_torch_saver(what_to_save)
         self._logger.torch_save()
 
-        self._logger.register_key('Metrics/EpRet', window_length=50)
-        self._logger.register_key('Metrics/EpCost', window_length=50)
-        self._logger.register_key('Metrics/EpLen', window_length=50)
+        self._logger.register_key('Metrics/EpRet', window_length=6)
+        self._logger.register_key('Metrics/EpCost', window_length=6)
+        self._logger.register_key('Metrics/EpLen', window_length=6)
 
```

Comment:
To avoid affecting the results of previous experiments, can we consider reading from the configs? You can perform it like:
```python
self._logger.register_key('Metrics/EpRet', window_length=self._cfgs._logger_cfgs.window_lens)
```